### PR TITLE
Jcm getting started

### PR DESCRIPTION
**Pull request by cmcfadzean**
## Description
Updates the info on the getting started page of the docs.

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
🔥  LGTM!

File: app/views/pages/getting_started.html.erb [hunk 1]
```diff
@@ -8,15 +8,66 @@ These reusable patterns are created from flexible building blocks we call compon
...
+    </div>
+  <% end %>
+  <%= sage_component SageGridCol, { breakpoint: :lg, size: "3" } do %>
+    <div class="docs-feature">
+      <i class="devicon devicon-react-original colored"></i>
+      <h4>React Components</h4>
+      <p>Our React components are used for situations where the desired solution for our customers is highly interactive or has lots of state management.</p>
+    </div>
+  <% end %>
+  <%= sage_component SageGridCol, { breakpoint: :lg, size: "3" } do %>
```
Comment (philschanely) [inline app/views/pages/getting_started.html.erb:37]: Consider switching this up before React components as these could be seen as being paired with the Rails stuff to ensure interactivity there.

File: app/views/pages/getting_started.html.erb [hunk 2]
```diff
@@ -8,15 +8,66 @@ These reusable patterns are created from flexible building blocks we call compon
...
+    <div class="docs-feature">
+      <i class="devicon devicon-javascript-plain colored"></i>
+      <h4>Vanilla Javascript</h4>
+      <p>When minor interactions are needed we lean on vanilla javascript to ensure that our code is simple and can stand the test of time.</p>
```
Comment (philschanely) [inline app/views/pages/getting_started.html.erb]: Consider: "Our Rails components have clean, minimal JavaScript needed to ensure the necessary interactions and accessibility features are available right out of the box."

File: app/views/pages/getting_started.html.erb [hunk 3]
```diff
@@ -8,15 +8,66 @@ These reusable patterns are created from flexible building blocks we call compon
...
+      <i class="devicon devicon-rails-plain-wordmark colored"></i>
+      <h4>Rails Components</h4>
+      <p>Our default approach to solving customer problems is via ROR. We use custom Rails Components to serve our system to the app.</p>
+    </div>
+  <% end %>
+  <%= sage_component SageGridCol, { breakpoint: :lg, size: "3" } do %>
+    <div class="docs-feature">
+      <i class="devicon devicon-react-original colored"></i>
+      <h4>React Components</h4>
+      <p>Our React components are used for situations where the desired solution for our customers is highly interactive or has lots of state management.</p>
```
Comment (philschanely) [inline app/views/pages/getting_started.html.erb]: Consider: "We build React versions of many of our components that mimic the same functionality. A few of our components exist only as React components due to their complexity."

File: app/views/pages/getting_started.html.erb [hunk 4]
```diff
@@ -8,15 +8,66 @@ These reusable patterns are created from flexible building blocks we call compon
...
+", use_sage_type: true) %>
+  <% end %>
+  <%= sage_component SageGridCol, { breakpoint: :lg, size: "5" } do %>
+    <%= image_pack_tag("docs/components.jpg", style: "display:block;max-width:500px;margin:0 auto;width:100%;height:auto", size: "1000x1036") %>
+  <% end %>
+<% end %>
+<hr>
+<%= md(" 
+
+This system is a work of shared ownership and we encourage any member of the Kajabi team to feel empowered to participate in its ongoing improvement. Understanding and improving the methodologies of the system is the key to the system's longevity. We encourage anyone planning on contributing to the system to first read over our Guides:
```
Comment (philschanely) [inline app/views/pages/getting_started.html.erb]: "methodologies of the system" I'm not sure what you mean. Consider removing or clarifying that sentence.

**Review by philschanely — APPROVED** [review]
A few suggestions to wording. Otherwise 👍

**Pull request merged**